tools/dmitool/dmitool.py

```diff
@@ -1,22 +1,23 @@
-""" Python 3.4 wrapper for dmitool.
+""" Python 2.7 wrapper for dmitool.
```

Comment:
Python 2.7 is deprecated.
if that´s the latest thing bay has, you should look at tg.